refinery/data_set_manager/tasks.py

```diff
@@ -192,10 +211,22 @@ def parse_isatab(username, public, path, identity_id=None,
                         # 3. Finally we need to get the checksum so that we can
                         # compare that to our given file.
                         investigation = ds.get_investigation()
-                        file_store_item = FileStoreItem.objects.get(
-                            uuid=investigation.isarchive_file
-                        )
-                        logger.info("Get file: %s", file_store_item)
+                        try:
+                            # this handling really should not matter since
```

Comment:
https://www.python.org/dev/peps/pep-0257/

Reply:
could you maybe clarify what you are referring to here? i think this is just a comment, not a docstring.  should we be using triple quotes for multiline?  there are other places in the code where we use the repeated hash.